### PR TITLE
docs: add kata-runtime to Docker daemon.json

### PR DESCRIPTION
**Pull request by morph027**
Fixes: #484

Signed-off-by: Stefan Heitmüller <stefan.heitmueller@gmx.com>

### CODE REVIEW

**Comment by morph027** [comment]
Hm, not sure why the Signed-off-by does not get noticed ....

**Comment by grahamwhaley** [comment]
Yeah, I'm going to take a peek at that - normally when we see that it is because the git email addr does not match the SOB email addr - I was going to check in git directly. It is a quirk of 'pullapprove' :-(

**Comment by grahamwhaley** [comment]
yeah, that is my best guess - your git email addr (morphs@....) is different from your SOB email (stefan.....@...) - that is what I think is upsetting pullapprove.

**Comment by grahamwhaley** [comment]
sadly, that is a little out of our control, and is handled by https://about.pullapprove.com/  I think maybe in the past @jodh-intel may even have opened an Issue for them asking if that is right, or if we can have a clearer error message etc. :-)

**Comment by morph027** [comment]
Nah, will adjust them

**Comment by morph027** [comment]
Nergh, need to improve my git mojo ;)

File: install/docker/centos-docker-install.md [hunk 1]
```diff
@@ -14,41 +14,58 @@
...
-   $ sudo docker run busybox uname -a
-   ```
+    ```bash
+    $ sudo docker run busybox uname -a
+    ```
 
-   The previous command shows details of the kernel version running inside the
-   container, which is different to the host kernel version.
+    The previous command shows details of the kernel version running inside the
+    container, which is different to the host kernel version.
```
Comment (intelkevinputnam) [inline install/docker/centos-docker-install.md]: different _from_ the host

**Comment by egernst** [comment]
@jodh-intel  - this is changing the install guide bash code AFAIU -- can we handle multiple options as this PR introduces?

**Comment by chavafg** [comment]
CI fails because OBS is really slow, problem has been around since last week :( 

```
Trying other mirror.
http://download.opensuse.org/repositories/home%3A/katacontainers%3A/release/CentOS_7/x86_64/qemu-vanilla-bin-2.11%2Bgit.e3050471ff-42.1.x86_64.rpm: [Errno 12] Timeout on http://provo-mirror.opensuse.org/repositories/home:/katacontainers:/release/CentOS_7/x86_64/qemu-vanilla-bin-2.11+git.e3050471ff-42.1.x86_64.rpm: (28, 'Operation too slow. Less than 1000 bytes/sec transferred the last 30 seconds')
Trying other mirror.
http://download.opensuse.org/repositories/home%3A/katacontainers%3A/release/CentOS_7/x86_64/qemu-vanilla-data-2.11%2Bgit.e3050471ff-42.1.x86_64.rpm: [Errno 12] Timeout on http://provo-mirror.opensuse.org/repositories/home:/katacontainers:/release/CentOS_7/x86_64/qemu-vanilla-data-2.11+git.e3050471ff-42.1.x86_64.rpm: (28, 'Operation too slow. Less than 1000 bytes/sec transferred the last 30 seconds')
Trying other mirror.
```

**Comment by nitkon** [comment]
I tried these steps but failed to restart docker.
```
# rm -rf /etc/systemd/system/docker.service.d/
# sudo mkdir -p /etc/systemd/system/docker.service.d/
# cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/kata-containers.conf
> [Service]
> ExecStart=
> ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime
> EOF
[Service]
ExecStart=
ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime
# sudo systemctl daemon-reload
#  sudo systemctl restart docker
Job for docker.service failed because the service did not take the steps required by its unit configuration.
See "systemctl  status docker.service" and "journalctl  -xe" for details.
[root@nitkon nitesh]# systemctl  status docker.service
● docker.service - Docker Application Container Engine
   Loaded: loaded (/usr/lib/systemd/system/docker.service; disabled; vendor preset: disabled)
  Drop-In: /etc/systemd/system/docker.service.d
           └─kata-containers.conf
   Active: failed (Result: protocol) since Tue 2018-07-24 15:44:48 IST; 5s ago
     Docs: https://docs.docker.com
  Process: 28681 ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime (code=exited, status=1/FAILURE)
 Main PID: 28681 (code=exited, status=1/FAILURE)
      CPU: 34ms

Jul 24 15:44:47 nitkon systemd[1]: docker.service: Main process exited, code=exited, status=1/FAILURE
Jul 24 15:44:47 nitkon systemd[1]: Failed to start Docker Application Container Engine.
Jul 24 15:44:47 nitkon systemd[1]: docker.service: Unit entered failed state.
Jul 24 15:44:47 nitkon systemd[1]: docker.service: Failed with result 'protocol'.
Jul 24 15:44:48 nitkon systemd[1]: docker.service: Service hold-off time over, scheduling restart.
Jul 24 15:44:48 nitkon systemd[1]: Stopped Docker Application Container Engine.
Jul 24 15:44:48 nitkon systemd[1]: docker.service: Start request repeated too quickly.
Jul 24 15:44:48 nitkon systemd[1]: Failed to start Docker Application Container Engine.
Jul 24 15:44:48 nitkon systemd[1]: docker.service: Unit entered failed state.
Jul 24 15:44:48 nitkon systemd[1]: docker.service: Failed with result 'protocol'.
```

File: install/docker/centos-docker-install.md [hunk 2]
```diff
@@ -14,41 +14,58 @@
 
 1. Install the latest version of Docker with the following commands:
 
-   > **Note:** This step is only required if Docker is not installed on the system.
+    > **Note:** This step is only required if Docker is not installed on the system.
```
[the remark this answers is not in the document]
Comment (morph027) [reply install/docker/centos-docker-install.md]: Sure!

**Comment by jodh-intel** [comment]
Yep - if you look at the Travis log:

```
Found 2 commits between commit 16db82e9c82e2a3113434413753c12d6fdbe8046 and branch master
ERROR: Commit 16db82e9c82e2a3113434413753c12d6fdbe8046: Failed to find subsystem in subject: "rhel daemon.json notes"
ERROR: checkcommits failed. See the document below for help on formatting
commits for the project.
https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format
```

But as @egernst says, to resolve this simply squash the 2nd (RHEL) commit into the first one as all these docs all need to be updated as an atomic unit in this case.

**Pull request merged**